labcodes/confetti-storybook
# CSYS-285: fix-card

**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-285

**How to test:**
- Go to the staging app at https://fix-card--confetti-storybook.netlify.app/
- Check if the `className` control is included in `Card` components (`FilledCard`, `OutlineCard` and `OutlineFilledCard`)
- Fill the `className` control with a `test-className` tag
- Inspect the page
- Run `document.querySelector('.test-className')` at the console and check if the element is present on the page
- Check if `Card` components can display with `isHorizontal` mode in mobile viewports
 
**Description of your solution:**
This PR publishes changes from branch `fix-card` from `confetti-ds` submodule. It allow horizontal cards in all media queries. It also includes a new `className` prop for `Card` components.
